share/ext/luajit/build_luajit.sh

```diff
@@ -215,101 +237,6 @@ case $1 in
 					cp -v bin/x86_64-macos/luajit-32 $PREFIX/bin/$CONF_TARGET/luajit-32
 					set +e
 		}
```

Comment:
We nowadays have a build_win32.bat which works great. And to leave no ambiguity, I removed this section.